### DISCUSSION TITLE
Option to show only filenames in playlist

### DISCUSSION THREAD

**Discussion by OldGeezer916** [smplayer-dev/smplayer#370] [Q&A]
Is there any way to to ignore any meta titles & just always just show filenames? I know if you uncheck to get info automatically it will show the filenames when you add. Once the file starts playing, if there is a title that is what shows. This would be fine if the meta was always the title, but often it is not. It can be anything from a website name to just random junk. MkvToolNix can take out a title when you mux a file, but only one at a time. I have looked for any program that can batch remove them, but no luck. I'm not big on command line. I would need a GUI.
![PLAYLIST](https://user-images.githubusercontent.com/74526703/153039474-282ef85f-89d7-4d75-a97e-2bf00b046b14.png)
.

**Comment by smplayer-dev** [comment]
Edit playlist.ini, look for `change_name` and set it to false.

**Reply by smplayer-dev** [reply]
I've just added an option to change it in preferences -> playlist in https://github.com/smplayer-dev/smplayer/commit/bf25340e467ee9214f8f47dc582978056850f9da

**Reply by OldGeezer916** [reply]
Editing the playlist config file was simple enough. Did the trick both in portable Windows & installed in Ubuntu. Afraid your instructions for adding the option in preferences is way above my head. Next time my son is over he could do it. Even if there was a setting option, I don't know if I would ever change it back. The filename works for me. 

Thanks so much for the reply. Now I can add one more reason why Smplayer is better than VLC. I have been on their support site & found a more than a dozen pages of people complaining about this & not any help from them.

**Comment by RSMPO** [comment]
I've had the same problem with newer versions of SMPlayer for several years. An older version (21.1.0 portable) displayed the Windows filename in the playlist (which is what I want). Newer versions (most recently 23.12.0 portable) continue to display the tag info. At different times I have tried both **un**checking the _**Display title name instead of filename**_ and also manually editing the playlist.ini file to **change_name=false**, but neither works - the Recent Files playlist still displays the title stored internally in the file. I've quit the program and restarted it after each change, but nothing seems to work. FYI, the SMPlayer title bar correctly displays the filename, only the playlist seems to display the tag info.

Is there anything else I can try, or is there something that you can do to fix this in a newer version?

**Reply by OldGeezer916** [reply]
I'm using 23.6 both on Linux Ubuntu & Windows 8 & 10. I've known 23.12 was available as a portable for Windows. With Linux, you only get updates when they make them available. I have the option for title name unchecked. I found a video that had a title when I checked it with MKVtoolNix. It did not show up in the playlist before or after I play it. I do most of my viewing in Ubuntu. Thanks for the warning about 23.12. SMplayer portable is such a small folder I usually don't delete older versions. Simple to go back. Installed on Ubuntu, there would be no way to go back. When it does show up in my updates, I'll just uncheck it.
